### ISSUE TITLE
Improvement of import for native .qea/qeax files for EA16

### ISSUE THREAD

**Issue by zittirone** [schmitze87/AAroN#5]
Hello,
thanks a lot for your efforts in this nice plugin.
I was trying to use the export in combination with EA 16.1 on an both qea and qeax files on commit 4.4.35
On both the import fails  with the following error message.

```
2024-08-30 13:54:32.126+0000 ERROR Could not process the results
java.util.concurrent.ExecutionException: java.lang.RuntimeException: org.sqlite.SQLiteException: [SQLITE_ERROR] SQL error or missing database (no such table: usys_system)
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at aaron.plugin.sparx.SparxImporter.importEAP(SparxImporter.java:80) ~[aaron-plugin-4.4.35.jar:?]
	at org.neo4j.kernel.impl.proc.GeneratedProcedure_sparxeap19085643866700.apply(Unknown Source) ~[?:?]
	at org.neo4j.procedure.impl.ProcedureRegistry.callProcedure(ProcedureRegistry.java:214) ~[neo4j-procedure-5.20.0.jar:5.20.0]
	at org.neo4j.procedure.impl.ProcedureViewImpl.callProcedure(ProcedureViewImpl.java:150) ~[neo4j-procedure-5.20.0.jar:5.20.0]
	at org.neo4j.kernel.impl.newapi.ProcedureCaller$ForTransactionScope.doCallProcedure(ProcedureCaller.java:281) ~[neo4j-kernel-5.20.0.jar:5.20.0]
	at org.neo4j.kernel.impl.newapi.ProcedureCaller.callProcedure(ProcedureCaller.java:183) ~[neo4j-kernel-5.20.0.jar:5.20.0]
	at org.neo4j.kernel.impl.newapi.AllStoreHolder.procedureCallWrite(AllStoreHolder.java:796) ~[neo4j-kernel-5.20.0.jar:5.20.0]
	at org.neo4j.cypher.internal.runtime.interpreted.CallSupport$.$anonfun$callReadWriteProcedure$1(CallSupport.scala:63) ~[neo4j-c
```

the funny thing is that the import works on eapx files and an qea file that has been generated from a 32bit EA version but not on a native 64bit qea/qeax file.
Could you help to improve the importer here ?

**Comment by schmitze87** [comment]
Hi thanks for using the plugin. 
Your error sounds a bit strange. I expect usys_system to be there even for an .qea or .qeax file.
Actually I'm using it on those files 95% of the time.
Unfortunately (for you) I'm on vacation and can't investigate on it until September 16th. 
But just one thing I recognised by reading your message on my phone. Looks like you are using version 4.4.35 in combination with neo4j 5.x. Version 4.4.35 is intended to be used with neo4j 4.4.x. Maybe the error is the result of some strange classpath problems.

**Comment by zittirone** [comment]
Hello
thank you  for your reply. I think there have been 2 issues. One has been that the EA project transfer from eap to qea cerated issues in usys_system table. creating a new EA project and importing via xml export/import solved this issue.
In addition there has been an issue in your class AAroN-CORE->src->main->java->aaron->sparx->processors->DiagramLinksProcessor.java
I had to filter out diagramGuid!=null  and connectorEdge != nulll
Both fixes togehter made it working using aaron-4.4.35 and neo4j 5.x
thanks.
